### PR TITLE
Remove `READ_EXTERNAL_STORAGE` from the following flavors: cmmb_kenya and surveillance_covid19_kenya

### PR DESCRIPTION
**Pull request by derickl**

### CODE REVIEW

File: src/cmmb_kenya/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="org.medicmobile.webapp.mobile">
+
+  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" tools:node="remove" />
+
+</manifest>
```
Comment (mrsarm) [inline src/cmmb_kenya/AndroidManifest.xml:8]: Please add newline at end of the file.

File: src/surveillance_covid19_kenya/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="org.medicmobile.webapp.mobile">
+
+  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" tools:node="remove" />
+</manifest>
```
Comment (mrsarm) [inline src/surveillance_covid19_kenya/AndroidManifest.xml:7]: Same.

**Review by mrsarm — APPROVED** [review]
LGTM, only add the new lines at the end of the files before merge, please.

File: src/cmmb_kenya/AndroidManifest.xml (hunk 1 above)
Comment (derickl) [reply src/cmmb_kenya/AndroidManifest.xml]: Done

**Pull request merged**